source/_statement.html.md

```diff
@@ -11,6 +11,13 @@ We’ve also made the service text as simple as possible to understand.
 
 [AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
 
+## How accessible this website is
+We know some parts of this website are not fully accessible:
+
+* the ‘upload a file’ button may cause issues for some users
+* the way that screen readers announce the different regions of our pages can be confusing
+* one page links to a PDF document that is not fully accessible
```

Comment:
Can we be more specific and say "our 'Memorandum of Understanding' is a PDF document that is not fully accessible" ? We don't just 'link' to a PDF, we own the PDF!